modules/backend/assets/less/layout/layout.less

```diff
@@ -22,6 +22,12 @@ body.no-select {
     cursor: default !important;
 }
 
+// Remove focus outline for mouse clicks, keep for keyboard navigation (Only needs to happen on Firefox)
+@supports (-moz-appearance: none) {
+    a:focus:not(:focus-visible) {
+        outline: none;
+    }
+}
```

Comment:
Let's move this to the bottom of the file under a section called `Browser specific` like the `Screen specific` section we have right now, and then recompile the assets with `php artisan mix:install` & `php artisan winter:util compile less`.